tau_bench/model_utils/api/sample.py

```diff
@@ -94,15 +93,15 @@ def execute(self, invocable_or_invokables: Callable[..., T] | list[Callable[...,
         if self.panic_on_first_model_error:
             if isinstance(invocable_or_invokables, Callable):
                 results = list(
-                    func_tools.map(
+                    functools.map(
```

Comment:
I didn't see func_tools defined so I think this is just fixing a typo?

Reply:
Oh I was missing a module. I will add it in a follow-up commit